solutions/search/elasticsearch-basics-quickstart.md

```diff
@@ -15,9 +15,7 @@ The code examples in this tutorial are in [Console](../../explore-analyze/query-
 
 ## Requirements [getting-started-requirements]
 
-You can follow this guide using any {{es}} deployment. If you already have a deployment up and running, you can skip ahead to the [first step](#getting-started-index-creation).
-
-If not, refer to [choose your deployment type](/deploy-manage/deploy.md#choosing-your-deployment-type) for your options. To get started quickly, you can spin up a cluster [locally in Docker](get-started.md):
+You can follow this guide using any {{es}} deployment. If you have a deployment setup ready, skip ahead to the [first step](#getting-started-index-creation). If not, refer [choose your deployment type](/deploy-manage/deploy.md#choosing-your-deployment-type) to see all deployment options. To get started quickly, spin up a cluster [locally in Docker](run-elasticsearch-locally.md):
```

Comment:
Do we need setup here?

```suggestion
You can follow this guide using any {{es}} deployment. If you have a deployment ready, skip ahead to the [first step](#getting-started-index-creation). If not, refer to [choose your deployment type](/deploy-manage/deploy.md#choosing-your-deployment-type) to see all deployment options. To get started quickly, spin up a cluster [locally in Docker](run-elasticsearch-locally.md):
```

Reply:
Sorry, I should've instead said, "Do we need _the word_ setup here?"

I'm only recommending removing the word "setup" because I think "If you have a deployment ready..." reads more smoothly and is the more common phrasing. I didn't mean to suggest removing the guidance entirely 🙃.